electrum/tests/test_util.py

```diff
@@ -79,7 +87,7 @@ def test_format_satoshis_plain_decimal(self):
 
     def test_format_satoshis_plain_to_mbtc(self):
         self.assertEqual("0.01234", format_satoshis_plain(1234, decimal_point=5))
-
+      
```

Comment:
please remove whitespace change

Reply:
I think it showing that because I hit backspace accidentally while making those changes and hit enter to keep the same spacing. The spacing is the same, there are no changes.

Reply:
This line was previously empty but now it contains some spaces. It is a minor thing but still annoying.

Reply:
Okay got it, thanks :)